src/page-multi-facility/FacilityRowPlaceholder.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useRef, useState } from "react";
+import useVisibility from "react-use-visibility";
+
+import Loading from "../design-system/Loading";
+import { Facility } from "./types";
+
+interface Props {
+  facility: Facility;
+  children: React.ReactElement;
+}
+
+const FacilityRowPlaceholder: React.FC<Props> = (props) => {
```

Comment:
Can we add a comment somewhere in this file that explains its purpose? I think future contributors to the project may not get it immediately.

Reply:
Yes good idea!